src/org/thoughtcrime/securesms/FullMsgActivity.java

```diff
@@ -37,6 +38,21 @@ protected void onCreate(Bundle state, boolean ready) {
     loadRemoteContent = Prefs.getAlwaysLoadRemoteContent(this);
     webView.getSettings().setBlockNetworkLoads(!loadRemoteContent);
 
+    // setBuiltInZoomControls() adds pinch-to-zoom as well as two ugly buttons;
+    // the latter are hidden with setDisplayZoomControls() again.
```

Comment:
maybe note that on-screen zoom controls are also deprecated and recommended to be disabled by API docs

Reply:
:joy: